misc/install.func

```diff
@@ -166,21 +192,57 @@ fi
 EOF
   chmod +x /usr/local/bin/apt-proxy-detect.sh
   fi
-  $STD apt-get update
-  $STD apt-get -o Dpkg::Options::="--force-confold" -y dist-upgrade
-  rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED
+  if grep -qEi "debian|ubuntu" /etc/os-release; then
+     $STD apt-get update
+     $STD apt-get -o Dpkg::Options::="--force-confold" -y dist-upgrade
+   # RPM-based systems
+   elif grep -qEi "fedora|centos|rocky|almalinux" /etc/os-release; then
```

Comment:
I would much rather we didn't support RHEL variants just yet.

Reply:
removed, comes with next change for rpm based systems in a seperate pr, maybe january